salesagility/SuiteCRM
# Fixed #9406 - Validation displayed static message isn't correct

Closes #9406 

As explained in the issue, there is an incorrect message displayed during the field validation of non-required fields.

## Description
This PR modifies the static message that appears during the validation of non-required fields of an EditView.

## Motivation and Context
The message can be confusing.

## How To Test This
Go to Contacts EditView. Paste this code in the Browser console:
```
addToValidateCallback("EditView", "phone_work", "integer", false, "Custom validation", function() {
    return false;
});
```
And Save the record. It should display the validation message in the "Phone Work" field: "Invalid value: Custom validation".

Note: You might need to empty caché after applying this patch.

## Types of changes

### Final checklist
